test/e2e/network/dns.go

```diff
@@ -380,7 +383,13 @@ var _ = SIGDescribe("DNS", func() {
 		validateTargetedProbeOutput(f, pod3, []string{wheezyFileName, jessieFileName}, svc.Spec.ClusterIP)
 	})
 
-	ginkgo.It("should support configurable pod DNS nameservers", func() {
+	/*
+		Release: v1.16
+		Testname: DNS, custom dnsConfig
+		Description: Create a Pod with DNSPolicy as None and custom DNS configuration, specifying nameservers and serach path entries.
+		Pod creation MUST be successful and provided DNS configuration MUST be configured in the Pod.
+	*/
+	framework.ConformanceIt("should support configurable pod DNS nameservers", func() {
 		ginkgo.By("Creating a pod with dnsPolicy=None and customized dnsConfig...")
 		testServerIP := "1.1.1.1"
```

Comment:
let me check if this will work for IPv6,
maybe we can use this https://github.com/kubernetes/kubernetes/blob/8e05e8346aaff7797bc4acaa94669915a8672b12/test/e2e/framework/networking_utils.go#L83

Reply:
the test seems to pass, but don't know if we should switch to an IPv6 address if the cluster is IPv6. I think that a simple if will be enough an easier to read since is a hardcoded value

```
testServerIP := "1.1.1.1"
if framework.TestContext.IPFamily == "ipv6 {
    testServerIP = "2001:beef::1"
}
```

Reply:
Seems good to me above suggestion but as @aojea mentioned, should we switch to an IPv6 address or not, remain in question. I would update code asap get clarity.

Reply:
I see. While addressing #80193 this should be taken care.